3-Register/src/app/layout.tsx

```diff
@@ -0,0 +1,31 @@
+import { Manrope } from "next/font/google";
+import Image from "next/image";
+import "./globals.css";
+
+const manrope = Manrope({
+  subsets: ["latin"],
+  weight: ["300", "400", "500", "600"],
+});
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html className="h-full w-full" lang="en">
+      <body
+        className={`flex flex-row h-full w-full bg-custom-red text-custom-gray ${manrope.className}`}
+      >
+        <Image
+          src="/RegisterIllustration.png"
+          alt="Ilustration of user signing in"
```

Comment:
I think in cases like this, where the image is only decorative and doesn't relate to any functionality, we should set an empty `alt` property.
```suggestion
          alt=""
```